general.css

```diff
@@ -0,0 +1,973 @@
+@font-face {
+    font-family: 'et-line';
+    src:url('et-line-font/fonts/et-line.eot');
+	src:url('et-line-font/fonts/et-line.eot?#iefix') format('embedded-opentype'),
+		url('et-line-font/fonts/et-line.woff') format('woff'),
+		url('et-line-font/fonts/et-line.ttf') format('truetype'),
+		url('et-line-font/fonts/et-line.svg#et-line') format('svg');
+	font-weight: normal;
+    font-style: normal;
+}
+
+[data-icon]:before {
+	font-family: 'et-line';
+	content: attr(data-icon);
+	speak: none;
+	font-weight: normal;
+	font-variant: normal;
+	text-transform: none;
+	line-height: 1;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+    display:inline-block;
+    font-size: 2rem; 
+}
```

Comment:
Maybe you could use the css file that you actually have in the et-line folder to not copy the same code here.